### PR TITLE
Remove extra densifies from freq code

### PR DESCRIPTION
**Pull request by jkgoodrich**

### CODE REVIEW

File: gnomad_qc/v4/annotations/generate_freq.py [hunk 1]
```diff
@@ -780,11 +845,12 @@ def generate_faf_grpmax(ht: hl.Table) -> hl.Table:
...
     )
+    faf_meta_expr = hl.flatten(faf_meta_expr)
     ht = ht.annotate_globals(
-        faf_meta=hl.flatten(faf_meta_expr),
-        faf_index_dict=[
-            make_faf_index_dict(hl.eval(x), label_delimiter="-") for x in faf_meta_expr
-        ],
+        faf_meta=faf_meta_expr,
+        faf_index_dict=make_freq_index_dict_from_meta(
+            faf_meta_expr, label_delimiter="-"
```
Comment (jkgoodrich) [inline gnomad_qc/v4/annotations/generate_freq.py:852]: Right now we have `_` for the freq_index_dict and `-` for the faf_index_dict, which should we use for both?

File: gnomad_qc/v4/resources/annotations.py [hunk 2]
```diff
@@ -283,7 +283,7 @@ def get_freq(
     :param finalized: Whether to return the finalized frequency table. Default is True.
     :return: Hail Table containing subset or overall cohort frequency annotations.
     """
-    ht_name = f"gnomad.exomes.v{version}"
+    ht_name = f"gnomad.exomes.v{version}.frequencies"
```
Comment (mike-w-wilson) [inline gnomad_qc/v4/resources/annotations.py:286]: ha

File: gnomad_qc/v4/annotations/generate_freq.py [hunk 3]
```diff
@@ -412,25 +412,65 @@ def generate_freq_ht(
...
+        non_ukb_strata_expr = [e for e in non_ukb_strata_expr if "downsampling" in e]
+
+        logger.info("Generating non-ukb downsampling group_membership array....")
+        non_uk_group_membership_ht = generate_freq_group_membership_array(
+            meta_ht,
+            non_ukb_strata_expr,
+            downsamplings=hl.eval(non_ukb_ds_ht.downsamplings),
+            ds_pop_counts=hl.eval(non_ukb_ds_ht.ds_pop_counts),
+        )
+        # Remove the first one because it is the indexing for the full subset.
```
Comment (mike-w-wilson) [inline gnomad_qc/v4/annotations/generate_freq.py:438]:
```suggestion
        # Remove the first entry because it is the index for the full subset.
```

**Review by mike-w-wilson — APPROVED** [review]
Just a couple small things but this checks out and LGTM! Thank you for getting this done so quickly ⚡

**Comment by mike-w-wilson** [comment]
The test isnt done running but it seems likely a checkpoint of the split VDS at L250 will remove our mem issues. If the tests are successful, I can commit to this branch or open a PR.

File: gnomad_qc/v4/annotations/generate_freq.py [hunk 4]
```diff
@@ -412,25 +412,65 @@ def generate_freq_ht(
...
+            )
+        )
+        # Remove the first two because they are adj and raw for the full subset.
+        group_membership_globals = group_membership_globals.annotate(
+            **{
+                g: group_membership_globals[g].extend(non_uk_globals[g][2:])
+                for g in ["freq_meta", "freq_meta_sample_count"]
+            },
+            non_ukb_downsamplings=non_uk_globals.downsamplings,
+            non_ukb_ds_pop_counts=non_uk_globals.ds_pop_counts,
```
[the remark this answers is not in the document]
Comment (jkgoodrich) [reply gnomad_qc/v4/annotations/generate_freq.py:455]: oops, nope, just in a hurry, thanks for the fix!

**Pull request merged**